logstash-configs/conf.d/00-input-elasticsearch-1w-param.conf

```diff
@@ -0,0 +1,10 @@
+input {
```

Comment:
please create a separate directory with configs for this job

Reply:
done